src/odemis/acq/milling/millmng.py

```diff
@@ -417,10 +419,9 @@ def _run_milling_tasks(self, feature: CryoFeature, workflow_task: MillingWorkflo
         self._future.set_progress()
 
         filename = self.get_filename(feature, "Milling-Tasks")
-        USE_OPENFIBSEM = False # TODO: add option to use openfibsem
-        if USE_OPENFIBSEM:
-            from odemis.acq.milling.openfibsem import run_milling_tasks_openfibsem
-            self._future.running_subf = run_milling_tasks_openfibsem(tasks=milling_tasks)
+        if is_fibsemos_available():
+            from odemis.acq.milling.fibsemos import run_milling_tasks_fibsemos
```

Comment:
I don't think it's necessary any more to import it inline. You should be able to place this import at the top of the file, with the rest of the imports, and it'll still import fine if fibsemOS is not available.